ruby/hamsters.rb

```diff
@@ -0,0 +1,35 @@
+puts "What is the hamster's name?"
+
+name = gets.chomp
+
+puts "Volume level (1-10)"
+
+volume = gets.chomp.to_i
+
+if volume == 0
+  volume = nil
+else
+end
+
+puts "Fur color"
+
+furcolor = gets.chomp
+
+puts "Good candidate for adoption? (y/n)"
+
+candidate = gets.chomp
+
+puts "Estimated age?"
+
+age = gets.chomp.to_i
+
+if age == 0
+  age = nil
```

Comment:
why did you choose to set age = nil if age == 0?

nil is a class object that represents a non value or a falsey state. I feel that a Hammy can get away with having an age of 0